src/actions/projectActions.ts

```diff
@@ -122,3 +122,47 @@ export function projectMonthUpload(file: File, type: 'timesheet' | 'inbound', pr
       .then(() => dispatch(busyToggle.off()));
   };
 }
+
+export function projectsMonthOverviewUpload(file: File, monthId: string) {
+  return (dispatch: Dispatch) => {
+    dispatch(busyToggle());
+    const req = request.put(buildUrl(`/attachments/project_month_overview/${monthId}/allTimesheets`));
+    req.attach(file.name, file);
+    req.then(response => {
+      dispatch({
+        type: ACTION_TYPES.PROJECTS_MONTH_OVERVIEWS_UPDATE,
+        projectsMonthOverview: response.body,
+      });
+      success(t('config.popupMessage'));
+      return true;
+    })
+      .catch(catchHandler)
+      .then(() => dispatch(busyToggle.off()));
+  };
+}
+
+export function deleteProjectsMonthOverview(id: string) {
+  return (dispatch: Dispatch) => {
+    dispatch(busyToggle());
+    const req = request.delete(buildUrl(`/attachments/project_month_overview/${id}/allTimesheets`));
+    req.then(response => {
+      dispatch({
+        type: ACTION_TYPES.PROJECTS_MONTH_OVERVIEWS_DELETE,
+        projectsMonthOverviewId: id,
+      });
+      success(t('config.popupMessage'));
+      return true;
+    })
+      .catch(catchHandler)
+      .then(() => dispatch(busyToggle.off()));
+  };
+}
+
+export function deleteProjectMonthAttachmentDetails(projectMonth: ProjectMonthModel) {
+  return (dispatch: Dispatch) => {
+    dispatch({
+      type: ACTION_TYPES.PROJECTS_MONTH_UPDATE,
+      projectMonth: {...projectMonth, attachments: []},
```

Comment:
Dit geeft geen probleem wanneer je de "timesheet" delete en er was ook al een "inbound invoice"?

Reply:
Deze functie wordt enkel opgeroepen nadat we een invoice hebben aangemaakt voor de projectmonth. Welk probleem zou dit volgens jou moeten geven? @Laoujin

Reply:
Ik doe een update van ProjectMonth. Ah er is al een action voor. Ik gebruik die action. Dat lijkt te werken.

En opeens zijn mijn attachments weg.